### PR TITLE
Add default casting option to ufunc

### PR DESCRIPTION
**Pull request by okuta**
This PR reduces Python function call.

### CODE REVIEW

File: cupy/core/elementwise.pxi [hunk 1]
```diff
@@ -840,4 +844,7 @@ cpdef create_ufunc(name, ops, routine=None, preamble='', doc=''):
         out_types = tuple([numpy.dtype(t).type for t in out_types])
         _ops.append((in_types, out_types, rt))
 
-    return ufunc(name, len(_ops[0][0]), len(_ops[0][1]), _ops, preamble, doc)
+    ret = ufunc(name, len(_ops[0][0]), len(_ops[0][1]), _ops, preamble, doc)
```
Comment (niboshi) [inline cupy/core/elementwise.pxi]: How about making it an argument of `ufunc.__init__()`?

Comment (okuta) [reply cupy/core/elementwise.pxi]: OK, I will fix

**Comment by niboshi** [comment]
jenkins, test this please

**Comment by niboshi** [comment]
LGTM!

**Pull request merged**